src/Map.js

```diff
@@ -37,6 +50,9 @@ let Map = {
         map.addControl(searchControl);
 
         map.on('click', e => this.notify('map-click', e.latlng));
+        map.on('draw:created', e => {
+            if (e.layerType === 'rectangle') this.notify('rectangle', e)
+        })
```

Comment:
Can you please an old man stuck in his ways and include a semi-colon?  It'll give me peace of mind.